SPEC.md

```diff
@@ -159,8 +159,7 @@ This example pod will use a set of three apps:
 
 #### Pod UUID
 
-The executor must assign an [RFC4122 UUID](http://www.ietf.org/rfc/rfc4122.txt) to each pod.
-This UUID is exposed to the pod through the [Metadata Service](#app-container-metadata-service).
+Each pod much be assigned an [RFC4122 UUID](http://www.ietf.org/rfc/rfc4122.txt). This UUID is exposed to the pod through the [Metadata Service](#app-container-metadata-service).
```

Comment:
I think we should err on the side of describing this more verbosely, if not necessarily prescriptively (I am anticipating more questions around the purpose of the UUID) - maybe something like:
"The UUID serves as a canonical reference to a pod within a given administrative domain.
In this context, an administrative domain is linked to the scope of the associated metadata service.
For example, given a metadata service that is federated across a geographical cluster of systems, the pod UUID is uniquely scoped to the same cluster."

..am I making things better or worse?